index.js

```diff
@@ -78,5 +83,182 @@ function loader (mcVersion) {
       return new Item(item.blockId, item.itemCount, item.itemDamage, item.nbtData)
     }
   }
+
+  Item.getEnchants = (item) => {
+    const normalize = (enchList) => enchList.map(ench => {
+      return { lvl: ench.lvl, name: mcData.enchantments[ench.id].name }
+    })
+    if (mcData.isOlderThan('1.13')) {
+      let itemEnch
+      if (item.name === 'enchanted_book' && item.nbt !== null) {
+        itemEnch = nbt.simplify(item.nbt).StoredEnchantments
+      } else if (item.nbt !== null) {
+        itemEnch = nbt.simplify(item.nbt).ench
+      } else {
+        itemEnch = []
+      }
+      return normalize(itemEnch)
+    }
+    const gg = nbt.simplify(item.nbt)
+    console.log(gg)
```

Comment:
remember to remove this

Reply:
>1.13.2 still doesnt work so it would break before that

Reply:
I meant the logging

Reply:
> I meant the logging

oh okay

Reply:
this is removed since the feature is added